firefox-ios/firefox-ios-tests/Tests/ClientTests/Mocks/MockWebKit.swift

```diff
@@ -117,3 +117,36 @@ class WKScriptMessageMock: WKScriptMessage {
         return overridenFrameInfo
     }
 }
+
+// MARK: - WKURLSchemeTaskMock
+
+/// Minimal fake WKURLSchemeTask used to capture callbacks.
+final class WKURLSchemeTaskMock: NSObject, WKURLSchemeTask {
```

Comment:
Can you check if we need to mark this as `@unchecked Sendable` with strict concurrency on?

Reply:
I don't see any warnings 😄